src/index.ts

```diff
@@ -382,7 +383,12 @@ function barfTurn() {
       prepWandererZone(),
       Macro.if_(
         `monsterid ${$monster`Angry ghost`.id}`,
-        Macro.skill("saucestorm").repeat()
+        Macro.trySkill("curse of weaksauce")
```

Comment:
could we incorporate a sing along?

Reply:
Sure, might as well add extract as well

Reply:
also given that this is the same PR I should perhaps name him